### PR TITLE
Enable collecting metrics from user

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?

Enable collect metrics from user

### Why are the changes needed?

so that user knows that they enabled short-circuit read from the related metrics.

### Does this PR introduce any user facing changes?

Web UI will show the correct user metrics

### CODE REVIEW

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -4411,7 +4411,7 @@ public String toString() {
           .build();
   public static final PropertyKey USER_METRICS_COLLECTION_ENABLED =
       new Builder(Name.USER_METRICS_COLLECTION_ENABLED)
-          .setDefaultValue(false)
+          .setDefaultValue(true)
```
Comment (maobaolong) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:4414]: This change will make the default behavior changed, the user report metric will harm for performance, do you really need to do this change while users do not aware this change?

**Review by maobaolong — COMMENTED** [review]
Left a question inline.

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (LuQQiu) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java:4414]: @apc999 @yuzhu

Comment (yuzhu) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java:4414]: so far we have not seen much perf degradation (maybe 5-10%)? do you have specific instances where this makes perf much worse? @maobaolong

Comment (yuzhu) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java:4414]: one reason we want to make this default true, is users get confused why there is no local read metric when they just use it out of the box. we want to make the default experience better.

**Pull request merged**